src/lib.rs

```diff
@@ -121,6 +121,22 @@ impl<H, N> Precommit<H, N> {
 	}
 }
 
+/// A Primary proposed block.
```

Comment:
maybe a doc comment mentioning that this is a broadcast of the last round's estimate.